pkg/sql/plan/build_test.go

```diff
@@ -488,6 +488,7 @@ func TestSingleTableSQLBuilder(t *testing.T) {
 
 		"values row(1,1), row(2,2), row(3,3) order by column_0 limit 2",
 		"select * from (values row(1,1), row(2,2), row(3,3)) a (c1, c2)",
+		"select * from nation where n_name like ? or n_nationkey > 10 order by 2 limit '10' for update",
```

Comment:
I assume this query we will select primary key out as well and lock it before returning?

Reply:
yes, but it will have some different when there has a limit